database/migrations/2016_03_13_110032_create_questions_table.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateQuestionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {   
+        // A question is identified by its id, event's id and the organization's id.
+        Schema::create('questions', function (Blueprint $table)
+        {   
+            $table->increments('id');
+            $table->integer('user_id')->unsigned()->index();
+            $table->foreign('user_id')->references('id')->on('users')
+            ->onDelete('cascade');
+            $table->integer('organization_id')->unsigned()->index();
+            $table->foreign('organization_id')->references('id')->on('organizations')->onDelete('cascade');
+            $table->integer('event_id')->unsigned()->index();
```

Comment:
yes, I guess there is no need for an organization column for what we are currently working with in the master branch. But I think there should exist a relation between 'organizations' and 'events' in the sense of an organization having many events. And in this case the column would be crucial for identification.